Composer/packages/extensions/adaptive-form/src/components/FieldLabel.tsx

```diff
@@ -32,7 +32,9 @@ const DescriptionCallout: React.FC<DescriptionCalloutProps> = function Descripti
         styles: { root: { width: '288px', padding: '17px 28px' } },
         onRenderContent: () => (
           <div>
-            <h3 style={{ fontSize: '20px', margin: '0', marginBottom: '10px' }}>{title}</h3>
+            <h3 style={{ fontSize: '20px', margin: '0', marginBottom: '10px' }} aria-label={title + '.'}>
```

Comment:
Can you explain why adding a `.` for SR is important?

Reply:
Without it, the SR runs the header and explanation together without a pause.